src/main/java/org/folio/des/service/JobUpdatesService.java

```diff
@@ -92,6 +92,12 @@ private boolean updateJobPropsIfChanged(Job jobExecutionUpdate, Job job) {
       job.setErrorDetails(jobExecutionUpdate.getErrorDetails());
       result = true;
     }
+    if (jobExecutionUpdate.getErrorDetails() == null &&
+            jobExecutionUpdate.getExportTypeSpecificParameters() != null &&
+            jobExecutionUpdate.getExportTypeSpecificParameters().getVendorEdiOrdersExportConfig() != null) {
```

Comment:
You can make your code more readible using:
 Objects.isNull(jobExecutionUpdate.getErrorDetails()) &&
 ObjectUtils.allNotNull(jobExecutionUpdate.getExportTypeSpecificParameters(), jobExecutionUpdate.getExportTypeSpecificParameters().getVendorEdiOrdersExportConfig())